source/Orchestrator/lib/common-orchestrator-construct.ts

```diff
@@ -514,17 +516,23 @@ export class OrchestratorConstruct extends Construct {
 
     reuseOrchLogGroup.overrideLogicalId(`ReuseOrchestratorLogGroup`);
 
-    return new CfnStack(this, 'NestedLogStack', {
+    const logStack = new NestedStack(this, 'NestedLogStack', {
       parameters: {
         KmsKeyArn: kmsKeyParm.stringValue,
         ReuseOrchestratorLogGroup: reuseOrchLogGroup.valueAsString,
       },
-      templateUrl:
-        'https://' +
+    });
+    const cfnStack = logStack.nestedStackResource as CfnResource;
+
+    cfnStack.addPropertyOverride(
+      'TemplateURL',
+      'https://' +
         Fn.findInMap('SourceCode', 'General', 'S3Bucket') +
         '-reference.s3.amazonaws.com/' +
         Fn.findInMap('SourceCode', 'General', 'KeyPrefix') +
-        '/aws-sharr-orchestrator-log.template',
-    });
+        '/playbooks/' +
```

Comment:
Should this be here? I think this template is not in the playbooks subdirectory. Also extra slash in path

Reply:
No it shouldn't, typo from copy pasting. Thanks for saving me some time on the deploy